content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -491,12 +491,15 @@ Download the [jwcrypto](https://pypi.org/project/jwcrypto) library,
 if you haven't installed it on your system.
 {{< /tip >}}
 
+The JWT authentication has 60 seconds clock skew, this means the JWT token will become valid 60 seconds earlier than
+its configured `nbf` and remain valid 60 seconds after its configured `exp`.
+
 For example, the command below creates a token that
-expires in 5 seconds. As you see, Istio authenticates requests using that token successfully at first but rejects them after 5 seconds:
+expires in 5 seconds. As you see, Istio authenticates requests using that token successfully at first but rejects them after 65 seconds:
 
 {{< text bash >}}
 $ TOKEN=$(python3 ./gen-jwt.py ./key.pem --expire 5)
-$ for i in $(seq 1 10); do curl --header "Authorization: Bearer $TOKEN" "$INGRESS_HOST:$INGRESS_PORT/headers" -s -o /dev/null -w "%{http_code}\n"; sleep 1; done
+$ for i in $(seq 1 20); do curl --header "Authorization: Bearer $TOKEN" "$INGRESS_HOST:$INGRESS_PORT/headers" -s -o /dev/null -w "%{http_code}\n"; sleep 5; done
```

Comment:
It looks like the test is now passing. The sample output in the box only shows 10 entries, not 20.  Maybe you go back to 10 loops and sleep 10. Then I would expect 7 200's and then 401's.

Reply:
sure, updated.